manifests/bosh-manifest/operations.d/600-users.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# Add os-conf release and users to the bosh director vm
+
+- type: replace
+  path: /releases/-
+  value:
+    name: os-conf
+    version: 21.0.0
+    url: https://bosh.io/d/github.com/cloudfoundry/os-conf-release?v=21.0.0
+    sha1: 7579a96515b265c6d828924bf4f5fae115798199
+
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value:
+    name: user_add
+    release: os-conf
+    properties:
+      users:
+      - name: mohameddeerow
```

Comment:
If we have to maintain the keys in two different places, can we add a test to check they're the same?